.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build
+on: [push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
I guess we need to install .NET?

Reply:
[.NET is pre-installed on the Ubuntu 20.04 images](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#net-core-sdk), so you should be good here.

[There's a GitHub Action you can use to install a specific .NET SDK version if required](https://github.com/actions/setup-dotnet), or you can run the entire action in one of the .NET container images if you want a stable environment.